src/mds/CInode.cc

```diff
@@ -1400,6 +1419,8 @@ void CInode::set_object_info(MDSCacheObjectInfo &info)
 void CInode::encode_lock_state(int type, bufferlist& bl)
 {
   ::encode(first, bl);
+  if (!is_base())
+    ::encode(parent->first, bl);
```

Comment:
This requires bumping `CEPH_MDS_PROTOCOL`?

Reply:
yes,